src/components/DataTable.js

```diff
@@ -1,40 +1,58 @@
 import React, { Fragment } from 'react';
 import { FormattedMessage, injectIntl } from 'react-intl';
-import { Input, Table, Spin, Alert } from 'antd';
+import { Input, Table, Spin, Alert, Row, Col, Button } from 'antd';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
 
 const Search = Input.Search;
 
-const DataTable = (props) => {
-  const { searchable, updateQuery, fetchData, data, query, loading, error, columns } = props;
+const DataTable = props => {
+  const { searchable, updateQuery, fetchData, data, query, loading, error, columns, noHeader, user } = props;
   const { q } = query;
-  const Header = loading ? <Spin size="small"/> : <FormattedMessage id="nResults"
-                                                                    defaultMessage={`{resultCount, number} {resultCount, plural,
+  const Header = loading ? <Spin size="small"/> :
+    <FormattedMessage
+      id="nResults"
+      defaultMessage={`{resultCount, number} {resultCount, plural,
         one {results}
         other {results}
       }
     `}
-                                                                    values={{ resultCount: data.count, q }}
-  />;
+      values={{ resultCount: data.count, q }}
+    />;
   const translatedSearch = props.intl.formatMessage({ id: 'search', defaultMessage: 'Search' });
+  // TODO probably, it'd better not to have such logic here
+  const createNew = () => {
+    props.history.push('create');
+  };
+
   return (
     <Fragment>
       {!error &&
       <div style={{ background: 'white', padding: 16 }}>
-        {searchable && <Search
-          placeholder={translatedSearch}
-          enterButton={translatedSearch}
-          size="large"
-          onChange={(e) => updateQuery({ q: e.target.value })}
-          value={q}
-          onSearch={val => fetchData({ q: val })}
-          style={{ marginBottom: '16px' }}
-        />
-        }
+        <Row>
+          {user && <Col span={4}>
```

Comment:
not any user can create - See also https://github.com/gbif/registry-console/issues/1

Reply:
Yes, I remember our discussion. I thought that I would do it after at least first item implementation. Could I do it after, let say, organizations?

Reply:
sure - no rush